Overview.src.html

```diff
@@ -4167,40 +4235,90 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 
   <p>To <span>process response</span> for <var title>response</var>, run these substeps:
 
   <ol>
    <li><p>If <var title>response</var>'s <span title=concept-response-type>type</span> is
    "<code title>error</code>", reject <var title>p</var> with a
-   <code title>TypeError</code>.
+   <code title>TypeError</code> and abort these steps.
 
-   <li><p>Otherwise, resolve <var title>p</var> with a new <code>Response</code> object
-   associated with <var title>response</var> and a new <code>Headers</code> object whose
+   <li><p>Let <var title>res</var> be a new <code>Response</code> object associated with
+   <var title>response</var> and a new <code>Headers</code> object whose
    <span title=concept-Headers-guard>guard</span> is "<code title>immutable</code>".
+
+   <li><p>If <var title>response</var>'s status is a <span>null body status</span>, resolve
+   <var title>p</var> with <var title>res</var> and abort these steps.
+
+   <li><p>Let <var title>pull</var> be an action that
+   <span title=concept-fetch-resume>resumes</span> the ongoing fetch if it is
+   <span title=concept-fetch-suspend>suspended</span>.
+
+   <li><p>Let <var title>cancel</var> be an action that
+   <span title=concept-fetch-terminate>terminates</span> the ongoing fetch with reason
+   <i title>end-user abort</i>.
+
+   <li><p>Let <var title>strategy</var> be an object. The user agent may choose any object.
+   <p class="note no-backref"><var title>strategy</var> is used to control the queuing strategy of
+   <var title>stream</var> constructed below.
+
+   <li><p>Set <var title>stream</var> to the result of
+   <span title=concept-construct-ReadableStream>constructing</span> a
+   <span title=concept-ReadableStream>ReadableStream</span> with <var title>strategy</var>,
+   <var title>pull</var> and <var title>cancel</var>. If that threw an exception, run the following
+   substeps:
+   <ol>
+    <li><p>Reject <var title>p</var> with that exception.
+
+    <li><p><span title=concept-fetch-terminate>Terminate</span> the ongoing fetch with reason
+    <i title>fatal</i>.
+   </ol>
+
+   <li><p>Otherwise, run the following substeps:
+   <ol>
+    <li><p>Set <var title>res</var>'s <span title=concept-Response-readable-stream>readable
+    stream</span> to <var title>stream</var>.
+
+    <li><p>Resolve <var title>p</var> with <var title>res</var title>.
+   </ol>
   </ol>
+  <p>To <span>process response body</span> for <var title>response</var>, run these substeps:
+  <ol>
+   <li><p>If <var title>stream</var> is null, abort these steps.
+
+   <li><p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> whose
+   contents are <var title>response</var>'s <span title=concept-response-body>body</span> into
+   <var title>stream</var>.
 
-  <p>To <span>process response body</span> for <var title>response</var>, do nothing.
+   <li><p>Clear out <var title>response</var>'s <span title=concept-response-body>body</span>.
```

Comment:
Hmm, that is [another TODO] (https://github.com/yutakahirano/fetch-with-streams/issues/40). In my plan I would do the refactoring after landing this change. Replacing &lt;concept-response-body&gt; will impact other specs in non-trivial ways (https://xhr.spec.whatwg.org/#dom-xmlhttprequest-send for example), do you want to do it here?